test/e2e/ion-autocomplete.single-select.e2e.html

```diff
@@ -8,47 +8,60 @@
     <title>ion-autocomplete end to end test page</title>
 
     <link rel="stylesheet" href="../../bower_components/ionic/css/ionic.css">
-    <link rel="stylesheet" type="text/css" href="../../src/ion-autocomplete.css"/>
+    <link rel="stylesheet" type="text/css" href="../../src/ion-autocomplete.css" />
 
     <script src="../../bower_components/ionic/js/ionic.bundle.js"></script>
     <script src="../../src/ion-autocomplete.js"></script>
     <script type="text/javascript">
         var app = angular.module('ion-autocomplete-sample', ['ionic', 'ion-autocomplete'])
-                .controller('IonAutocompleteController', function ($scope) {
```

Comment:
Same here: Please revert the formatting as this adds unneeded changes to the pull request. (for the whole block from line 17 to 42)